packages/cspell-eslint-plugin/fixtures/dictionaries/eslint.config.mjs

```diff
@@ -0,0 +1,41 @@
+// @ts-check
+import eslint from '@eslint/js';
+import { defineCSpellConfig, defineCSpellPluginOptions } from '@cspell/eslint-plugin';
+import cspellRecommended from '@cspell/eslint-plugin/recommended';
+
+/**
+ * @type { import("eslint").Linter.Config[] }
+ */
+const config = [
+    eslint.configs.recommended,
+    cspellRecommended,
+    {
+        ignores: ['eslint.config.mjs'],
+    },
+    {
+        files: ['**/*.js'],
+        rules: {
+            '@cspell/spellchecker': [
+                'warn',
+                defineCSpellPluginOptions({
+                    debugMode: false,
+                    autoFix: false,
+                    cspell: defineCSpellConfig({
+                        dictionaryDefinitions: [
+                            {
+                                name: 'custom-dict',
+                                supportNonStrictSearches: false,
+                                // cspell: words Codeco
+                                words: ['IPv4', 'IPv6', 'Codeco'],
+                                suggestWords: ['codeco->Codeco'],
+                            },
+                        ],
+                        dictionaries: ['custom-dict'],
```

Comment:
@JounQin,

This is an example on how `supportNonStrictSearches` can be used.

Reply:
Thanks for the guidance so much! I'll take a deeper look soon later.

Reply:
@Jason3S I tried it at https://github.com/alauda/doom/pull/96/files#diff-669cc760bf8a381ee744b98e01d00e2c2a710f7f7acbc29c8058d9d7dfed526aR69-R79, but it seems not working as expected, [`ipv6`](https://github.com/alauda/doom/pull/96/files#diff-def3cadd0d84d5753fda1ea1e5bff2844e2b46375d79b41be1aba0ff812eeb15R51) is not reported.

Reply:
It doesn't look like you are using the setting `supportNonStrictSearches`.

```
    dictionaryDefinitions.push({
      name: dictionary,
      path: `.cspell/${file.name}`,
      addWords: true,
      supportNonStrictSearches: false // Add this line to disable lower case matches.
    })
```

Reply:
The issue with IPV4 is that `ipv` is in the AWS dictionary. So the only way to avoid it is to turn off the AWS dictionary or add it to the forbid list like you did.

Reply:
It is also necessary to set `minWordLength: 3`.